README.md

```diff
@@ -27,65 +27,55 @@ interfaces in pure Python.
     src="https://mybinder.org/badge_logo.svg"/>
 </a>
 
-# Install Django IDOM
+# Quick Example
 
-```bash
-pip install django-idom
-```
+## `example_app/components.py`
 
-# Django Integration
+This is where you'll define your [IDOM](https://github.com/idom-team/idom) components. Ultimately though, you should
+feel free to organize your component modules you wish. Any components created will ultimately be referenced
+by Python dotted path in `your-template.html`.
 
-To integrate IDOM into your application you'll need to modify or add the following files to `your_project`:
+```python
+from idom import component, html
+from django_idom import IdomWebsocket
 
-```
-your_project/
-├── __init__.py
-├── asgi.py
-├── settings.py
-├── urls.py
-└── example_app/
-    ├── __init__.py
-    ├── components.py
-    ├── templates/
-    │   └── your-template.html
-    └── urls.py
-```
```

Comment:
This one is a bit hard for me. I like the example repository structure, but also, this repository structure example was made off of my personal preferred folder structure. There are a lot of Django repos out there that structure things significantly differently.

The only thing in here that **must** remain consistent (based on Django structure) is the `example_app` directory structure.

How would you feel if I restructured this to focus on `example_app`?

Reply:
Maybe we could split this up into two lists of files - those in `example_app` and those which are standard Django files. That way you still get a high level picture of what needs to be modified, but you can non-prescriptive about where the standard Django files belong.

Reply:
I don't think it's too important to highlight the standard Django files. Django repos expect readers to have an understanding of Django filenames.

For example,
[django-silk](https://github.com/jazzband/django-silk#installation)
[django-bootstrap](https://github.com/zelenij/django-bootstrap-v5#installation)
[django-downloadview](https://github.com/jazzband/django-downloadview#example)
[django-split-settings](https://github.com/sobolevn/django-split-settings#usage)

The main reason I removed this block is since we aren't dependent on specific file names/locations, we can get by without an example structure. An end user is expected to already have knowledge on everything besides `components.py`. And I think our text block describing `components.py` is adequate enough to cover that.

The second reason is we are definitely packing way too much information into the readme, even in its now shortened state. When the additional hooks or helper functions get developed, we're going to want to think about having some real docs for this repo.

Reply:
Got it. One last option would be to make the section headings link to any relevant Django docs?

```markdown
# [asgi.py](https://docs.djangoproject.com/en/4.0/howto/deployment/asgi/)
```

Reply:
And agreed on making "real docs" for this. I could set up the scaffolding for the docs as some point, but I don't think I'll have any time to write them up myself unfortunately.

Reply:
> Got it. One last option would be to make the section headings link to any relevant Django docs?
> 
> ```gfm
> # [asgi.py](https://docs.djangoproject.com/en/4.0/howto/deployment/asgi/)
> ```

Yeah that's a good idea, I'll toss that in to this PR.